### PR TITLE
Documentation update, MQTT reconnect logging

### PR DESCRIPTION
**Pull request by Jalle19**
@tomrosenback

### CODE REVIEW

**Comment by Jalle19** [comment]
For some reason I can't add you as reviewer, GitHub just says "Nothing to show" :shrug:

**Comment by tomrosenback** [comment]
I haven't got any permissions to this repo, that is why.

File: docs/CONNECTION.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* https://doc.enervent.com/op/op.ViewOnline.php?documentid=999&version=1 (Freeway WEB-väyläsovitin - Asennus- ja käyttöohjeet)
+* https://web.archive.org/web/20201020102005/http://ala-paavola.fi/jaakko/doku.php?id=pingvin
+
+Please read the above documents before continuing!
+
+## Requirements
+
+* A 4P4C cable (sometimes called RJ10)
+* An RS-485 device
```
Comment (tomrosenback) [inline docs/CONNECTION.md:13]: *knitpicking*
A RS-485 device

File: README.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ https://www.home-assistant.io/integrations/switch.rest/ with minimal effort. See
 
 * Node.js 12.x or newer
 * An Enervent ventilation unit with EDA automation (Pingvin and Pandion confirmed working)
-* An RS-485 device (e.g. `/dev/ttyUSB0`) connected to the Enervent unit's Freeway port
+* An RS-485 device (e.g. `/dev/ttyUSB0`) connected to the Enervent unit's Freeway port (see
```
Comment (tomrosenback) [inline README.md:24]: *knitpicking*
A RS-485 device

**Review by tomrosenback — APPROVED** [review]

File: docs/CONNECTION.md (hunk 1 above)
Comment (Jalle19) [reply docs/CONNECTION.md:13]: AFAIK no, this is correct. Since it's pronounced "arr ess 485" it should be with an "an".

Comment (tomrosenback) [reply docs/CONNECTION.md:13]: Ah, refreshing my grammars here :D Thank you for the lesson :D

**Pull request merged**